tutorials/python/python-video-generation/code/python_video.py

```diff
@@ -0,0 +1,52 @@
+import moviepy.editor as mpy
+import gizeh as gz
+
+
+def render_text(t):
+    BLUE = (59/255, 89/255, 152/255)
```

Comment:
Good Catch! I'll update this.

> Adding type annotations will be a plus, but I'm not sure if this can confuse people who are not used to use types in Python. What do you think?

It's not a bad idea but this post is meant for people with a basic Python knowledge and that extra information can be confusing as you mention.